src/web/components/SideBar.tsx

```diff
@@ -0,0 +1,121 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';  // Import the Link component
+import { FaHome, FaChartBar, FaBook, FaCode, FaDatabase, FaCloud, FaUserGraduate, FaUsers } from 'react-icons/fa';
+
+// Menu items with their respective icons and routes
+const menuItems = [
+    { name: 'Home', icon: <FaHome />, route: '/' },
+    { name: 'FinOps hubs', icon: <FaChartBar />, route: '/hubs' },
+    { name: 'Power BI', icon: <FaBook />, route: '/power-bi' },
+    { name: 'PowerShell', icon: <FaCode />, route: '/powershell' },
+    { name: 'Bicep modules', icon: <FaDatabase />, route: '/bicep-registry' },
+    { name: 'Open data', icon: <FaCloud />, route: '/open-data' },
+    { name: 'Learning', icon: <FaUserGraduate />, external: true, route: 'https://learn.microsoft.com/en-us/cloud-computing/finops/toolkit/finops-toolkit-overview' }, // External link
```

Comment:
Make sure you don't include locales ("en-us") in links. But in this case, can we create a dedicated page for this? We don't have one today, but we can copy the learning section of the home page as a starting point. It's okay if that happens in a separate PR.

Also, we should probably put learning resources first since you should learn about FinOps before you use all the tools that automate it 🤔 (I know the mockup had it last. Sorry about that.)
```suggestion
    { name: 'Learning', icon: <FaUserGraduate />, external: true, route: 'https://learn.microsoft.com/cloud-computing/finops/toolkit/finops-toolkit-overview' }, // External link
```

Reply:
When you say put learning resources first do you mean the order of things on the sidebar? So under home comes learning, then the tools?